help.html

```diff
@@ -275,35 +278,64 @@ <h2 id="hello">Hello, World!</h2>
         asking the micro:bit to scroll the words across the LED screen and then display an image of a heart.</p>
     <p>Why not try making the micro:bit scroll your name?</p>
    
-    <div class="hide-blocks">
-    <h2 id="blockly">Blockly</h2>
+    <div >
+    <h2 id="fs">File system</h2>
 
-    <p>Use blockly to make new Python code from easy to assemble blocks of
-    code.</p>
+    <p>Use the file system to add or remove python modules, data logs and other file types
+        from the micro:bit. Selecting the load button in the menu will bring up the files modal window.
 
-    <p><img src="static/img/blocks.png"
-        class="image img-responsive center-block img-rounded"
-        alt="An animation demonstrating how to use snippets."/></p>
+    <p><img src="static/img/file-system.png"
+        class="image img-responsive center-block img-rounded" style="max-width: 75%;"
+        alt="modal dialogue for the file system interface"/></p>
 
-    <p>Blockly updates Python in the code editor as you move blocks around.
-    There are lots of different blocks to cover all the features of the
-    micro:bit. You can make more advanced code by combining blocks together
-    (such as in the example above).</p>
-
-    <p>You should use Blockly to create <em>new code</em> since moving blocks
-    around will overwrite any existing code you have in the editor (and you may
-    lose work as a result). The arrangement of blocks isn't (currently)
-    saved, so Blockly makes a great way to make simple Python code without
-    lots of typing. In the end, you'll learn that simply typing Python is by
-    far the quickest and easiest way to write code.</p>
+    <p>When the micro:bit is flashed with MicroPython, it enables a simple file system to store and retrieve files from the device</p>
```

Comment:
At the moment we cannot retrieve files from the device.

Reply:
I've edited the wording here to reflect this